runtime/testapps/src/main/java/com/google/apphosting/runtime/jetty9/senderrorapp/SendErrorServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.apphosting.runtime.jetty9.senderrorapp;
+
+import java.io.IOException;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
```

Comment:
You will need two versions of this, one for javax API and one for jakarta API.

Then you can reference them in the required `web.xml` files.

Reply:
Done